src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -134,6 +134,9 @@
 #include "global/realfn.h"
 #include "engraving/iengravingconfiguration.h"
 
+#include <QRegularExpression>
+#include <QRegularExpressionMatch>
```

Comment:
I believe you should be using `std::regex` and `std::regex_match` rather than the Qt stuff

Reply:
You'd still need to remove these includes

Reply:
done